### PR TITLE
Switch all code.google.com dependencies to their github

### PR DESCRIPTION
**Pull request by rafrombrc**
equivalents.

### CODE REVIEW

File: cmake/externals.cmake [hunk 1]
```diff
@@ -168,15 +168,16 @@ if (INCLUDE_DOCKER_PLUGINS)
 endif()
 
 if (INCLUDE_MOZSVC)
-    add_external_plugin(git https://github.com/mozilla-services/heka-mozsvc-plugins 848fc1f3aa858472150c7af5463661393d3c4f3b)
+    #add_external_plugin(git https://github.com/mozilla-services/heka-mozsvc-plugins 848fc1f3aa858472150c7af5463661393d3c4f3b)
+    add_external_plugin(git https://github.com/mozilla-services/heka-mozsvc-plugins :local)
```
Comment (trink) [inline cmake/externals.cmake]: left over from testing?

Comment (rafrombrc) [inline cmake/externals.cmake]: yup, good catch. i'll push my commit to heka-mozsvc-plugins repo and will get this fixed.

Comment (rafrombrc) [inline cmake/externals.cmake]: fixed.

**Comment by trink** [comment]
Do you want to switch the heka-mozsvc-plugins dependenciest too?

**Comment by rafrombrc** [comment]
I did, unless I missed something...

**Comment by trink** [comment]
No, I missed it... just merged it

**Pull request merged**